src/app/submission/edit/submission-edit.component.ts

```diff
@@ -112,6 +121,8 @@ export class SubmissionEditComponent implements OnDestroy, OnInit {
             this.notificationsService.info(null, this.translate.get('submission.general.cannot_submit'));
             this.router.navigate(['/mydspace']);
           } else {
+          const { errors } = submissionObjectRD.payload;
+          this.submissionErrors = parseSectionErrors(errors);
```

Comment:
Tiny thing...these two lines are not aligned properly.  They should be indented two spaces so that it's clear they are within the `else` clause

Reply:
done